dt-core/admin/components/setup-wizard-keys.js

```diff
@@ -0,0 +1,209 @@
+import {
+  html,
+  css,
+} from 'https://cdn.jsdelivr.net/gh/lit/dist@2.4.0/all/lit-all.min.js';
+import { OpenLitElement } from './setup-wizard-open-element.js';
+
+export class SetupWizardKeys extends OpenLitElement {
+  static styles = [
+    css`
+      :host {
+        display: block;
+      }
+    `,
+  ];
+
+  static get properties() {
+    return {
+      step: { type: Object },
+      firstStep: { type: Boolean },
+      _options: { type: Object },
+      saving: { type: Boolean },
```

Comment:
does this need to have `attribute: false`?
otherwise it counts as a component attribute

Reply:
good suggestion